optional-modifications/README.md

```diff
@@ -14,28 +13,38 @@ the desired goal.
 
 ## How to use patches
 
-The patches are designed mostly to help modify the existing
-configuration files. You will need to run the `install.sh` script
-afterwards.
+The patches are designed mostly to help modify the existing configuration files. You will need to run the `install.sh` script afterwards.
 
-They should be run from the root directory. For example, the
-`external-kafka` patches should be run as:
+They should be run from the root directory. For example, the `external-kafka` patches should be run as:
```

Comment:
Can we ensure patches work fine without conflicts as we change `docker-compose.yml` over time?

Maybe a GHA or something like that?

I'll whip up a PR if you're OK with it.

Reply:
Done that here https://github.com/getsentry/self-hosted/pull/3793